gulpfile.js

```diff
@@ -118,6 +128,33 @@ gulp.task('build-ci', ['clean'], function (cb) {
     runSequence('build', cb);
 });
 
+gulp.task('build-test', function buildTest(cb) {
+    var config = _.defaultsDeep({}, require('./webpack.config'));
+
+    config.entry = {};
+    config.entry[path.join('request', 'repository', 'request-repository-cache')] = './test/request/repository/request-repository-cache.spec.ts';
```

Comment:
Can you add a comment here mentioning that the `path.join` bit for the key in the entry causes the entry bundle to be put in that folder structure?